spring-cloud-azure-samples/eventhubs-binder-sample/src/main/java/com/example/SourceExample.java

```diff
@@ -30,4 +31,9 @@ public String postMessage(@RequestParam String message) {
         this.source.output().send(new GenericMessage<>(message));
         return message;
     }
+
```

Comment:
What's this?

Reply:
easy way to check wheter the app starts successfully in App Service, instead of using curl, user can request this via browser